src/Autofac/Features/Scanning/OpenGenericScanningRegistrationExtensions.cs

```diff
@@ -13,7 +13,7 @@ namespace Autofac.Features.Scanning;
 /// <summary>
 /// Helper methods to assist in scanning registration.
 /// </summary>
-internal static partial class ScanningRegistrationExtensions
+internal static class OpenGenericScanningRegistrationExtensions
```

Comment:
Having this just One Big Class was causing some of the confusion with what was consuming what.